src/data/defaultMetricAnnotation.md

```diff
@@ -0,0 +1,7 @@
+This is a starter template for a metric annotation. You can add whatever
+freeform markdown you think might be helpful to someone looking at this metric:
+this could be a set of notes, links to dashboards or reports which use this
+metric, or whatever you like! Bear in mind that the Glean Dictionary is visible
+to the public, please do not share information about Mozilla's partners which is
+under NDA. It's ok to put in links that require SSO or authentication, but
+please mark them as such.
```

Comment:
@irrationalagent What do you think of this?

Reply:
It may be useful to include a default metadata section at the top of this document to encourage the use of tags and metadata.

Reply:
Perhaps the best place for extended instruction on the process here is a `CONTRIBUTING.md` in the glean-annotations repo. GitHub should present a call-out to `CONTRIBUTING.md` in the PR interface for new contributor, so it would get some exposure even in the case of editing an existing annotation, rather than just proposing a new annotation.

This template text can encourage the user to read `CONTRIBUTING.md` if it's their first addition.

Reply:
@acmiyaguchi yeah, we talked about this a bit yesterday in the product instrumentation meeting yesterday. An earlier draft did contain such tags, but I opted to leave them out for a first draft while we collect a few initial annotations from DS to prove out this feature. Likely we would want to enforce a strong schema on this metadata and I think that's a whole set of PRs / additional work.

👍 on creating a CONTRIBUTING.md in glean-annotations. That seems easier to keep up to date than an obscure template file in this repo. I'll do something up there and update the text here to reference it.

Reply:
Hmm yea. I think this is ok, given that most people who will be contributing to this understand what should and should not be shared publically. At the worst, we will have a PR to review things - though that does not exactly prevent information from being visible in the commit. 

In addition to the NDA point, we generally try NOT to put absolute values or counts from telemetry into public facing-documentation. We are less strict about percentages and relative changes. But I'm wondering if its worth adding some guidance about not putting "actual" telemetry numbers, be they absolute or relative, into the metadata.

Reply:
Some of these points would be more in scope for https://github.com/mozilla/glean-annotations/pull/6, now that we've moved most of the guidance over there. I'll try to incorporate this feedback in that place.